install-panel.sh

```diff
@@ -41,6 +41,12 @@ if ! [ -x "$(command -v curl)" ]; then
   exit 1
 fi
 
+if ! [ -x "$(command -v tput)" ]; then
```

Comment:
Does `tput` not come as "standard" for any of the supported distros?

Reply:
Yes it does I think. Will remove